### ISSUE TITLE
[Suggestion] Disclaimer Notice

### ISSUE THREAD

**Issue by Retrolovania** [Monika-After-Story/MonikaModDev#10056]
Just a thought regarding curbing the amount of unhealthy usage of MAS.

Some people have suggested that there should be a disclaimer (perhaps in the TS warning section) that clarifies the truth of the game; that it's not a fully authentic relationship and that IRL partnerships are more valuable. This is not to deter the goal of MAS, but rather to help alleviate that part of the userbase that may have the potential to take it too far.

TL;DR Something like the disclaimer for Postal 2, but for MAS and about the truth of the relationship as opposed to violence in video games.

**Comment by dreamscached** [comment]
Personally I don't think it's MAS' problem, but whoever plays it. I think the percentage of those that would *really* give up a person they could possibly have a relationship with is <1% and even then, it's their own choice.

**Comment by Retrolovania** [comment]
Given it's enough of an amount to cause annoyment within the general community, I still think it has some worth looking into. I also don't think that the small amount in general makes it totally worth excluding as a potential idea, either. Gotta keep everyone in mind, after all. (And it's not something as divisive as religious events, I'd say.)

**Comment by dreamscached** [comment]
I don't see how can you make it either not immersion-breaking or not canon-breaking. Monika wants to be player's partner, it won't look good at all if we backtracked and made her like

> Oh um, sorry player, change of heart... I think you should find a girlfriend in the real world, there's no point in being here

Alternatively, what do you want, a message like splash saying

> hey buddy you spent an entire week here, maybe go touch some grass?

I don't get how do you even see it.

**Comment by Retrolovania** [comment]
That's why I suggested putting it in the warning you see when you launch the mod for the very first time. Monika is outside of that.

**Comment by dreamscached** [comment]
Personally I haven't seen much drama involving this outside Reddit, but alright, I can believe that. Still — these people have concerns over something that isn't their business. Violence in games, although *very unlikely*, but can sparkle up some real violence, that will actually harm people. What do we have here? No one is harmed.

**Comment by Retrolovania** [comment]
Regardless, I still believe the addition would be beneficial. Would there be any harm in adding it as opposed to not? It's not breaking any sort of immersion if it's put in the one place that has no immersion.

**Comment by dreamscached** [comment]
I personally believe it's just unnecessary and isn't worth spending time on. Nobody is really adding a warning

> You're about to play a very Gorey and violent game, do not ever do that in real life

because generally you're expected to understand that clearly yourself. Plus I don't support the narrative either, it's people's choice what to do, not everyone wants a real relationship, some maybe be afraid of them, have no time for them etc. It's not up to anyone to decide if it's wrong or not.

**Comment by Retrolovania** [comment]
Are people expected to already understand that? This is the first time I'm hearing this, to be honest.
Frankly I don't understand why there's any hostility towards the idea; all it would amount to is an additional sentence at the very beginning of the mod and that's all. Just a disclaimer that the mod's not meant to be taken that seriously by those that play it.
